content/post/kubedb-v2023.02.28/index.md

```diff
@@ -0,0 +1,181 @@
+
+From this release, client certificates secret for TLS enabled databases now contain combined PEM certificates. 
+The combined PEM certificate is stored under the key `tls-combined.pem` in the client certificate secret.
+To enable this, the cert-manager needs to be deployed with feature-gate `AdditionalCertificateOutputFormats=true`. 
+To install cert-manager with this flag using helm you can use the following command.
+```bash
+ helm install \
+   cert-manager jetstack/cert-manager \
+   --namespace cert-manager \
+   --create-namespace \
+   --set installCRDs=true \
+   --set featureGates=AdditionalCertificateOutputFormats=true
+```
+## Acme Protocol Certificate
+
+### ProxySQL & PgBouncer
+In this new release, we introduced TLS support using ACME Protocol based Certificates with Let’s Encrypt. 
+Now you can provision KubeDB PgBouncer with TLS secured connection via both CA and ACME issuer using cert-manager.
+
+# PostgreSQL
+
+Now you will be able to migrate from StandAlone to High Availibility Cluster for  PostgreSQL. The migration process will copy the data for new replicas first and thus preventing possible data loss during the migration process.
+
+```bash
+apiVersion: ops.kubedb.com/v1alpha1
+kind: PostgresOpsRequest
+metadata:
+  name: move-to-high-availibility
+  namespace: demo
+spec:
+  type: HorizontalScaling
+  horizontalScaling:
+    replicas: 3
+    streamingMode: Synchronous
+    standbyMode: Hot
+  databaseRef:
+    name: demo-pg
+```
+
+## ElasticSearch
+Lastest versions for  OpenSearch  `2.0.1` and `2.5.0` added in this release. Now, you can provision and manage `OpenSearch V2` with `Elasticsearch` CRD using KubeDB.
+Check out all the currently supported Opensearch Versions using the following command.
+```bash
+$ kubectl get esversion | grep opensearch
+opensearch-1.1.0            1.1.0     OpenSearch     opensearchproject/opensearch:1.1.0             29h
+opensearch-1.2.2            1.2.2     OpenSearch     opensearchproject/opensearch:1.2.2             29h
+opensearch-1.3.2            1.3.2     OpenSearch     opensearchproject/opensearch:1.3.2             29h
+opensearch-2.0.1            2.5.0     OpenSearch     opensearchproject/opensearch:2.0.1             29h
+opensearch-2.5.0            2.5.0     OpenSearch     opensearchproject/opensearch:2.5.0             29h
+```
+
+If you are using any of the Opensearch 1.x.x versions, you can very easily upgrade to `OpenSearch 2.x.x` with data persistence using the following `ElasticsearchOpsRequest` CRD.
+
+
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: ElasticsearchOpsRequest
+metadata:
+  name: os-upgrade
+  namespace: demo
+spec:
+  type: Upgrade
+  databaseRef:
+    name: os-cluster                             # refer the name of you opensearch instance
+  upgrade:
+    targetVersion: opensearch-2.0.1    # refer the target version to be upgraded 
+```
+We recommend upgrading to OpenSearch `2.0.1` before upgrading to any other V2 minor versions.
+
+### Dashboard
```

Comment:
Opensearch Dashboards ?

Reply:
either add heading with ### Latest version release/support or remove the heading for dashboard